src/lib/import-export/import/importers/sql-importer.ts

```diff
@@ -0,0 +1,8 @@
+import { BackupContents } from '../types';
+import { DefaultImporter } from './Importer';
+
+export class SqlImporter extends DefaultImporter {
```

Comment:
As those importers don't differ, and we don't know if they will, should we just use `DefaultImporter` in all cases and avoid extending the base one?

Reply:
Done! :smile: